### ISSUE TITLE
[Catalog] Clicking a system in catalog leads to 404

### ISSUE THREAD

**Issue by iamEAP** [backstage/backstage#4695]
Clicking a system in the catalog leads to a 404

## Expected Behavior

It'd be great to see a nice system overview page!  Or not be linked at all, yet.

## Current Behavior

A brief blip of the outline of a page, followed by an entity not found page takeover.

## Possible Solution

- If a system page exists, let's link it / use it (or if we have a mock/comp for such a page, let's implement it!)
- If we don't have such a page yet, we should at least un-link the system in the catalog table.

## Steps to Reproduce

![404'd](https://user-images.githubusercontent.com/3496491/109126405-7db2f780-774d-11eb-91fd-e61dc6fce546.gif)

## Your Environment
https://demo.backstage.io

**Comment by Fox32** [comment]
I guess the data is not fully imported into the demo, there is a separate `all-systems.yaml` and also `all-domains.yaml` that have to be[ registered in the demo.](https://github.com/backstage/demo/blob/master/app-config.yaml#L46-L61). The new glob import might help here too. Afterwards we might still need to configure the domain and system entity page with the right widgets.

**Comment by Fox32** [comment]
I wonder if we either need a label "demo" or if we should move that ticket to the demo repository. This is not a bug in the default app of Backstage.

**Comment by adamdmharvey** [comment]
Yep; might well do for transition to https://github.com/backstage/demo - I think @OrkoHunter let's curate that, but not 100% sure.
